src/core/agent/services/identifier.types.ts

```diff
@@ -27,13 +27,13 @@ interface IdentifierDetails extends IdentifierShortDetails {
   signifyOpName?: string;
   s: string;
   dt: string;
-  kt: number;
+  kt: string | string[];
```

Comment:
note in case you're wondering why this can be `string[]` - the threshold can be weighted, so it will use fractions like `["1/2", "1/2", "1/1"]`. We don't support this for now but OK to update this now :)

Curious though - can we use `Pick` in TS to take these fields from `State` in Signify-TS? That would be better